src/test/resources/gradle/liberty.gradle.test.wrapper.app/build.gradle

```diff
@@ -0,0 +1,44 @@
+apply plugin: 'liberty'
+apply plugin: 'war'
+
+version '1.0'
+group 'liberty.gradle.test.wrapper.app'
+
+sourceCompatibility = 17
+targetCompatibility = 17
+tasks.withType(JavaCompile) {
+    options.encoding = 'UTF-8'
+}
+
+
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+        dependencies {
+        classpath 'io.openliberty.tools:liberty-gradle-plugin:3.2'
```

Comment:
can we update to the latest version of the LGP, which is `3.6.1`?

Reply:
will do